packages/gamut-labs/src/brand/LandingPageSection/LandingPageHero.tsx

```diff
@@ -58,6 +62,7 @@ export const LandingPageHero: React.FC<LandingPageHeroProps> = ({
   ctaHref,
   imgSrc,
   imgAlt,
```

Comment:
This is a little cleaner & easier to read IMO than a separate `defaultProps`:

```suggestion
  imgAlt = '',
```